src/widget.js

```diff
@@ -231,6 +231,46 @@ function pageVisited(change) {
 	});
 }
 
+function initialize(params) {
+	for (const method in params) {
+		switch(method) {
+			case 'customFied':
+				const { key, value, overwrite } = params[method];
+				setCustomField(key, value, overwrite);
+				continue;
+			case 'setCustomFields':
```

Comment:
Your code looks good, but here we need to ensure that is a valid `params[method]` to perform the `foreach`.

Reply:
I have added the checks
thanks :)

Reply:
Perfect, good job.

Reply:
thanks :)